### DISCUSSION TITLE
Whitelisted sites, but no green-colored entries in cleanup logs

### DISCUSSION THREAD

**Discussion by dlc2001** [Cookie-AutoDelete/Cookie-AutoDelete#1024] [Q&A / Support] [answered]
Hi, I am trying to diagnose a problem I'm having with CAD and to do this, I did a cleanup (Clean, including open tabs) and then looked at the log.  The examples in the FAQs and documentation show red entries for cookies that are being cleaned, and green for entries that are being kept.  I am getting red entries for things that are being cleaned, but no green entries.  I *do* have whitelisted sites, and there is evidence that CAD (or something) did "keep" these entries, since e.g. I am still logged in to the site.  But... shouldn't there be a green entry?  Somewhere in the documentation it also says that sometimes CAD is not in control and something else is instead... so I would prefer to see the green entries to confirm that CAD is in charge/in control of those sites' data.  Is there some reason why I do not see green entries?

The documentation also shows a filter for on the log - none, clean or keep - but I'm not seeing that set of choices on my logs.  Perhaps that is my problem?  Thank you.

**Comment by kennethtran93** [comment, marked as answer]
It's also noted on there that it needs some adjustments on it before we make it available again.  It's not too high on our priority list at this time, but the info in the doc is from a previous version.

**Reply by dlc2001** [reply]
Thank you, but I am confused by your response.  You say it's also "noted on there that it needs some adjustments on it" before you make it available again.  Are you saying that the green "kept" entries are not currently available because some adjustments are needed on this function?  Or are you talking about the documentation?  I brought up the documentation only because I wanted to demonstrate that I had looked before asking, and because I wondered if I had accidentally filtered off the entries I wanted to see.  

Your other response refers to cookies deleted by something else, but I am asking about the opposite... whitelisted entries.  I am trying to understand why I do not see kept entries in the logs for items that were demonstrably kept.  CAD did the right thing- it kept them - but there is no record of this in the logs.  Are you saying that the green entries not currently an available feature of CAD?  

Thank you.

**Reply by kennethtran93** [reply]
Apparently so. I only came onboard as a developer near the beginning of version 3, so i had left the code as is and fixed other bugs and added features as they come in. Guess that will be one of the next items I look into.

If you want to see the kept items, you can enable the debug option and follow the instructions within. There will be a lot of messages, being debug, but that would be one place you could see what was happening in near real time.  This includes tab changes and cookie changes. During cleanups, it would also print out what was kept/cleaned and its reason. This debug only lasts for the current browser session so you need to copy paste it if you want to keep any of it.

**Comment by kennethtran93** [comment]
Keep in mind that even though we may not have removed it, something else may have and that will not appear inside our logs.
